src/Bridges/Constraint/zero_one.jl

```diff
@@ -0,0 +1,110 @@
+"""
+    ZeroOneBridge{T}
+
+The `ZeroOneBridge` splits a `F`-in-`MOI.ZeroOne` constraint into a `F`-in-`MOI.Integer` constraint,
+a `F`-in-`MOI.LessThan(1)` constraint, and a `F`-in-`MOI.GreaterThan(0)` constraint.
```

Comment:
Why not just an `SingleVariable-in-Interval` constraint?

What happens if there are existing variable bounds? We might have to add a `ScalarAffineFunction-in-Interval` instead.

Reply:
> What happens if there are existing variable bounds? We might have to add a `ScalarAffineFunction-in-Interval` instead.

Just to make sure I understand, this is because `ZeroOne` is absent from the list of sets for which only one SingleVariable bound can be imposed per set 
e.g. 
https://github.com/JuliaOpt/MathOptInterface.jl/blob/0b5cd4d01965044e62483dd7bac0c86aa88d21c5/src/constraints.jl#L196-L203
but in bridging a `SingleVariable-in-ZeroOne` constraint to (among others) a `SingleVariable-in-LessThan` we could get a `MOI.UpperBoundAlreadySet` error?

Reply:
> Why not just an `SingleVariable-in-Interval` constraint?

I agree, let's keep as most structure as possible.

> What happens if there are existing variable bounds? We might have to add a `ScalarAffineFunction-in-Interval` instead.

Indeed, this might be errors but having a binary variables with bounds is a bit silly anyway. The bounds are either useless or force the value to either zero or one hence the variable can be removed.
If the bridge create a ScalarAffineFunction-in-Interval then it would destroy important structure.

> Just to make sure I understand ...

yes, exactly